coins.json

```diff
@@ -966,8 +966,8 @@
     "hrp": "bluzelle",
     "explorer": {
       "url": "https://bigdipper.net.bluzelle.com/",
```

Comment:
can you remove `/` from `url`. but add `/` to `txPath`, `accountPath`

Reply:
This is done.